config/tag2link.yml

```diff
@@ -0,0 +1,60 @@
+# THIS FILE IS AUTOGENERATED WITH THE script/misc/update-tag2link
+# PROGRAM DO NOT MANUALLY EDIT IT
+
+---
+3dmr: https://3dmr.eu/model/$1
+contact:facebook: https://www.facebook.com/$1
```

Comment:
Looking at https://taginfo.openstreetmap.org/keys/contact%3Afacebook#values - it seems to be incorrect